engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/migration/MigrationExecutionDto.ftl

```diff
@@ -0,0 +1,49 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/migration/execute-migration/index.html -->
+<#macro dto_macro docsUrl="">
+<@lib.dto>
+    
+    <@lib.property
+        name = "migrationPlan"
+        type = "object"
+        dto = "MigrationPlanDto"
+        additionalProperties = true
```

Comment:
I think this is wrong. There is only a single instance of `MigrationPlanDto`. A property of type `object` with `additionalProperties` set to `true` will be converted to a Java Map.

I think it should be the following:
```suggestion
        type = "ref"
        dto = "MigrationPlanDto"
```

You can check out the `ProcessInstanceSuspensionStateAsyncDto.ftl` file, I think we have the same case there.